tensor2tensor/models/research/universal_transformer.py

```diff
@@ -363,8 +363,8 @@ def update_hparams_for_universal_transformer(hparams):
   hparams.daisy_chain_variables = False  # Breaks multi-gpu in while loops.
 
   # If not None, mixes vanilla transformer with Universal Transformer.
```

Comment:
If you want to avoid usage of None, probably this comment needs to be updated as well?

Reply:
I.e., while technically true, sounds like you are trying to dissuade the user from using None